subsys/mgmt/suitfu/src/suitfu_mgmt_suit_image_fetch.c

```diff
@@ -211,7 +211,6 @@ int suitfu_mgmt_suit_missing_image_upload(struct smp_streamer *ctx)
 
 		session->last_notify_ts = k_uptime_get();
 
-		component_unlock();
```

Comment:
To my understanding - the idea was to unlock the component while external, possibly long operations are performed (i.e. writing data into the storage).